build/images/base/Dockerfile

```diff
@@ -21,6 +21,23 @@ RUN set -eux; \
     mkdir -p /opt/cni/bin; \
     wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
 
+FROM golang:1.13 as whereabouts-bin
```

Comment:
I am thinking should we just include whereabouts in the cni-binaries. Let us see what @antoninbas thinks.

Reply:
Yes, that makes it much simpler. I had updated Dockerfile accordingly and pushed the changes.
I will work with whereabouts community to add binaries as part of their release cadence.